codeforboston/maple
# Migrating scroll tracker to use bootstrap / styled components

# Summary

Migrating scroll tracker to use bootstrap / styled components (#1453 )

# Checklist

- [na] On the frontend, I've made my strings translate-able.
- [na] If I've added shared components, I've added a storybook story.
- [na] I've made pages responsive and look good on mobile.

# Screenshots

N/A (No visible changes - exactly the same behavior as production)

# Known issues

N/A

# Steps to test/reproduce

1. Go to the home page
1. Scroll down the page
1. See that the parallax effect works correct in the Hero Header and the About Section
